make/autoconf/flags.m4

```diff
@@ -265,6 +265,11 @@ AC_DEFUN_ONCE([FLAGS_PRE_TOOLCHAIN],
     fi
   fi
 
+  if test "x$OPENJDK_TARGET_OS" = xmacosx &&
+      test "x$OPENJDK_TARGET_CPU" = xaarch64; then
```

Comment:
Thank you for getting on this!
For symmetry, and to enable cross compilation on both directions, can we add the necessary flag for x64 too? 

Also, I doubt it matters much, but we could also only set these flags if COMPILE_TYPE=cross.

Reply:
Thinking again, it doesn't hurt to be explicit about this. Better always set these flags to avoid surprises.

Reply:
Cool, I haven't thought it is possible, although clearly it should be :) Thanks! Fixed by always setting the flag. I also rely on Github Action to check this with Xcode 11.3.1, it seems to be fine https://github.com/AntonKozlov/jdk/runs/2256008843